smarty/sampler.py

```diff
@@ -385,9 +406,10 @@ def AddAlphaSubstituentAtom(self, atom1type, bondset, atom2type):
         just be adding $(*zw) where z is the selected bond type and w is atom2type. It should raise an exception 
         if an alpha substituent is attempted to be added to an atom1type which already has an alpha substituent.
         """
-        print atom1type[0]
-        print atom2type[0]
-        print bondset[0]
+        if self.verbose: 
```

Comment:
I think they are helpful to have in general when we are tracking which atoms are being added. It might be worth leaving them for now.